packages/utils/src/common/types.ts

```diff
@@ -102,3 +102,5 @@ export interface SetLike<V> extends ReadonlySetLike<V> {
   clear(): void;
   delete(value: V): boolean;
 }
+
+export type Falsy = false | 0 | -0 | 0n | '' | null | undefined;
```

Comment:
As a result, I think it's best to completely remove it

```suggestion
```

Reply:
Removed now